Customer.io project/components/Sidebar.tsx

```
@@ -114,46 +124,49 @@ export const Sidebar: React.FC<SidebarProps> = ({
             </h3>
             <div className="mt-2 space-y-1">
               {menuItems.map((item, index) => (
                 <React.Fragment key={item.name}>
                   <button
                     onClick={() => onInsertBlock(item.name.toLowerCase())}
-                    className="flex items-center space-x-3 w-full p-2 text-left hover:bg-blue-50 rounded-lg group"
+                    className="flex items-center space-x-3 w-full p-2 text-left hover:bg-blue-50 rounded-lg group  hover:dark:!text-black"
                   >
-                    <div className="w-8 h-8 bg-gray-100 rounded-lg flex items-center justify-center group-hover:bg-blue-100">
-                      <span className="text-gray-600 group-hover:text-blue-600">
+                    <div className="w-8 h-8 bg-gray-100 rounded-lg flex items-center justify-center group-hover:bg-blue-100  hover:dark:!text-black">
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)